src/aks-preview/azext_aks_preview/vendored_sdks/azure_mgmt_preview_aks/v2022_03_02_preview/aio/operations/_managed_clusters_operations.py

```diff
@@ -417,6 +417,7 @@ async def list_cluster_user_credentials(
         resource_name: str,
         server_fqdn: Optional[str] = None,
         format: Optional[Union[str, "_models.Format"]] = None,
+        namespace_name = None,
```

Comment:
It's not a good idea to modify the SDK manually. If this feature is written in swagger, the automatically generated SDK would include the changes. Otherwise, these changes **will be erased in the next SDK update** (expected to update to 04-02-preview in just a few days).

Reply:
I removed the changes here and added the namespace sdk generated from the namespace swagger. The namespace swagger is separate from the AKS swagger. Regenerating the AKS sdk will not remove the namespace sdk, will it?

Reply:
That's OK. If the namespace service has python SDK officially released, may use it without vendor the code.

Reply:
cc @norshtein for awareness of modifying the `get-credentials` command.